server_py/flatgov/common/tasks.py

```diff
@@ -0,0 +1,10 @@
+from celery.task.schedules import crontab
+import datetime
+from django.utils import timezone
+from celery.decorators import periodic_task
+from .biden_statements import load_statements
+
+
+@periodic_task(run_every=(crontab(minute=0, hour=0), name="scrape-biden-statements-once-a-day", ignore_result=True)
```

Comment:
celery `@periodic_task` decorators are deprecated
See https://stackoverflow.com/a/56071240/628748

I'll move this to celery.py